### DISCUSSION TITLE
Video DownloadHelper translation - pt_BR

### DISCUSSION THREAD

**Discussion by giancagentile** [aclap-dev/video-downloadhelper#964] [Language & Translation]
[pt_BR.txt](https://github.com/user-attachments/files/16083240/pt_BR.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
